### PR TITLE
Reference to selector widgets.mdx

### PR DESCRIPTION
**Pull request by jas-kas**
Closes https://github.com/getsentry/sentry-docs/issues/8996

### CODE REVIEW

**Review by michellewzhang — APPROVED** [review]
thanks for adding! 
gif

**Review by c298lee — APPROVED** [review]

File: docs/product/session-replay/replay-page-and-filters.mdx [hunk 1]
```diff
@@ -16,7 +16,7 @@ You can search for or browse replays of user sessions on the **Replay** page, wh
 - **Dead Clicks**: User clicks on `a` and `button` tags that do not result in any page activity after 7 seconds (i.e. no HTML was added, removed, or updated; no visual changes were observed in the page).
 - **Rage Clicks**: Five or more clicks on a dead element (it exhibits no page activity after 7 seconds.) Rage clicks are a subset of dead clicks.
 
-SDK version `7.60.1` or higher is required to see rage and dead click data on the Replays page.
+The **Replay** page also shows two widgets titled "Most Dead Clicks" and "Most Rage Clicks" that show the top selectors that have been rage or dead clicked on. Expanding the selector will show example replays that contain clicks on that selector. You can also click the "See all selectors" button to see all selectors that have been rage or dead clicked on. SDK version `7.60.1` or higher is required to see rage and dead click data on the **Replay** page.
```
Comment (jas-kas) [inline docs/product/session-replay/replay-page-and-filters.mdx:19]:
```suggestion
The **Replay** page also shows two widgets titled "Most Dead Clicks" and "Most Rage Clicks" that displays the top selectors that have been rage or dead clicked on. Expanding the selector will show example replays that contain clicks on that selector. You can also click the "See all selectors" button to see all selectors that have been rage or dead clicked on. SDK version `7.60.1` or higher is required to see rage and dead click data on the **Replay** page.
```

Comment (lizokm) [inline docs/product/session-replay/replay-page-and-filters.mdx:19]:
```suggestion
The **Replay** page also has two widgets titled "Most Dead Clicks" and "Most Rage Clicks" that show the selectors with the most rage or dead clicks. Expanding the selector will show example replays where that selector was clicked. You can also click the "See all selectors" button to view all selectors that have gotten rage or dead clicks. SDK version `7.60.1` or higher is required to see rage and dead click data on the **Replay** page.
```

**Review by lizokm — APPROVED** [review]
Few more small tweaks, but looks good. Thanks for adding!!

**Pull request merged**